nautobot/cloud/models.py

```diff
@@ -162,3 +162,32 @@ class Meta:
 
     def __str__(self):
         return f"{self.cloud_network}: {self.prefix}"
+
+
+@extras_features(
+    "custom_links",
+    "custom_validators",
+    "export_templates",
+    "graphql",
+    "webhooks",
+)
+class CloudService(PrimaryModel):
```

Comment:
Maybe add `ordering=["name"]`, this should fix the test failure you had where you had to specify `ordered=False`

https://github.com/nautobot/nautobot/pull/5884/files#diff-0dafb14130323a8401ee7be55944247d1fc81dddb3f880bcbb0089cb069dd34dR201